### DISCUSSION TITLE
Can Sizif fit in a Spectrum Plus case? Or any other recommandation?

### DISCUSSION THREAD

**Discussion by JohnFante22** [UzixLS/zx-sizif-512#62] [General]
I have my Sizif in a rubbercase - taken from an Omni. 

I am getting a bit tired of typing on the rubber "keys" and therefore, I am considering "upgrading" to a Spectrum Plus case or another alternative keyboard.

Sizif fit nicely into a rubber case, but have anyone any experience with putting it into a Plus case? Or any other case for that matter?

**Comment by pgo-sk** [comment]
![IMG_20250608_095741](https://github.com/user-attachments/assets/600f89ae-670a-4a08-9f85-5d7e50e5f41f)

![IMG_20250608_095752](https://github.com/user-attachments/assets/9994c13e-567b-4c2c-92a1-ede8d0c1c16f)
128K+2 needs a bit of moving the 8pin keyboard connector to left with cable extension and a bit of new holes. otherwise works like a charm and even gives the comfort of opening the "hood" without disconnecting the keyboard cables
